examples/globe_vector.html

```diff
@@ -29,14 +29,44 @@
                     display: none;
                 }
             }
+            .couponcode:hover .coupontooltip {
```

Comment:
Where do these "coupon" comes from?

Reply:
They are used to display the feature’s properties

Reply:
Yes I know :-) I was asking about the meaning of "coupon": it doesn't seem to make sense in our context. These classes should be renamed.